### PR TITLE
upgraded ServiceStack and Eventstore packages

### PR DESCRIPTION
**Pull request by RustyChain**
Hi there,

I've just upgraded the the nuget packages for the latest one, the unit test are find, so hope you'll accept my pull request.
Let me know if you have any question.

Thanks,

Rusty

### CODE REVIEW

File: src/ServiceStack.EventStore/Dispatcher/EventDispatcher.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public EventDispatcher()
                     var request = new BasicRequest();
                     request.Headers.AddAll(headers);
 
-                    await HostContext.ServiceController.ExecuteAsync(typedEvent, request);
+                    await HostContext.ServiceController.ExecuteAsync(typedEvent, request,false);
```
Comment (DavidBrower) [inline src/ServiceStack.EventStore/Dispatcher/EventDispatcher.cs]: Can I ask what is the purpose of setting applyFilters to `false` here?

**Comment by DavidBrower** [comment]
Hi @RustyChain the PR looks fine. I've just added one question about the change to `EventDispatcher`?

File: src/ServiceStack.EventStore/Dispatcher/EventDispatcher.cs (hunk 1 above)
Comment (RustyChain) [inline src/ServiceStack.EventStore/Dispatcher/EventDispatcher.cs]: It was a mistake. I've reviewed and changed to true in the new commit. Thanks for the note :)

**Review by RustyChain — COMMENTED** [review]
applyFilters changed to true

**Comment by DavidBrower** [comment]
Hi @RustyChain looks fine. I'll merge in the changes.

**Pull request merged**